sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -122,26 +114,7 @@ export class MessageSender extends LinkEntity<AwaitableSender> {
         );
       }
       if (sender && !sender.isItselfClosed()) {
-        if (!this.isConnecting) {
-          log.error(
```

Comment:
With this change, we are losing out on a log statement for when `sender_close` event gets fired with no error. I would recommend updating the log statement a few lines above to remove the `if (senderError)` check on log the statement to denote that `sender_close` event occurred. 

Same for `session_close` in this file. 
Same for `receiver_close` and `session_close` events in streamingReceiver.ts file

Reply:
Done.